Product/Production/Common/Properties/src/main/resources/AdapterAuthenticationProxyConfig.xml

```diff
@@ -1,35 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns="http://www.springframework.org/schema/beans"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.5.xsd"
+       xsi:schemaLocation="
+        http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.5.xsd"
```

Comment:
The spring-beans-2.5.xsd is changed to 3.0 in other places. Any specific reasons not changed?